edb/server/protocol/auth_ext/oauth.py

```diff
@@ -97,7 +94,7 @@ async def _handle_identity(self, user_info: data.UserInfo) -> data.Identity:
             query="""\
 with
   iss := <str>$issuer_url,
-  sub := <str>$provider_id,
+  sub := <str>$provider_name,
```

Comment:
This should be `provider_id` still actually: it's the ID of the user in the provider's system not our "provider ID". Maybe just change it to `subject`?

Reply:
I thought we dropped `provider_id`?

Reply:
no, this is literally just a poorly-named variable for the `sub`/`subject` in this query. "subject's ID at the provider's service" = "provider_id". I think `subject` is the correct name now.

Reply:
Oh right, yeah it's just a variable name. Looks like it caught up in a find/replace. 🤦‍♂️

Reply:
yep! it was poorly named before anyway.